src/js/components/AuthorPicker.js

```
@@ -2,261 +2,129 @@
-                <Flex justify="space-between" align="center">
-                    <FlexItem>
-                        {selectedAuthorIds.length > 0 && (
-                            <Button
-                                variant="tertiary"
-                                onClick={() => onChange([])}
-                                className="apb-reset-btn"
-                            >
-                                {__('Clear All', 'author-profile-blocks')}
-                            </Button>
-                        )}
+                        <SelectControl
```

ESLint

[error] Replace `························` with `↹↹↹↹↹↹`

(prettier/prettier)